packages/appstore/src/components/modals/modal-manager.tsx

```diff
@@ -1,20 +1,18 @@
 import React from 'react';
 import { observer } from 'mobx-react-lite';
 import { ResetTradingPasswordModal } from '@deriv/account';
-import {
-    JurisdictionModal,
-    CFDPasswordModal,
-    CFDDbviOnBoarding,
-    CFDResetPasswordModal,
-    CFDTopUpDemoModal,
-    MT5TradeModal,
-    CFDPasswordManagerModal,
-    CompareAccountsModal,
-} from '@deriv/cfd';
 import MT5AccountTypeModal from './account-type-modal';
 import RegulatorsCompareModal from './regulators-compare-modal';
 import { useStores } from 'Stores';
 import CFDServerErrorDialog from '@deriv/cfd/src/Containers/cfd-server-error-dialog';
+import JurisdictionModal from '@deriv/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal';
+import CFDPasswordModal from '@deriv/cfd/src/Containers/cfd-password-modal';
+import CFDDbviOnBoarding from '@deriv/cfd/src/Containers/cfd-dbvi-onboarding';
+import CFDResetPasswordModal from '@deriv/cfd/src/Containers/cfd-reset-password-modal';
+import CFDTopUpDemoModal from '@deriv/cfd/src/Containers/cfd-top-up-demo-modal';
+import MT5TradeModal from '@deriv/cfd/src/Containers/mt5-trade-modal';
+import CFDPasswordManagerModal from '@deriv/cfd/src/Containers/cfd-password-manager-modal';
+import CompareAccountsModal from '@deriv/cfd/src/Containers/compare-accounts-modal';
```

Comment:
These are part of our entry in the webpack in cfd. Can't you use that ?
Importing files directly this way is not a good way of using components from other packages.

Reply:
Yes, this is fine for the components which we are using inside the `cfd` package, but since there are some components which we are using in the appstore package which throws error when we import it from `cfd` package because they cannot detect the path alias of the `cfd` package

Reply:
This one is not because of import aliases, The problem is that we shouldn't build the component and then use it in another package, That package should build the component instead.

As a temporary solution, we can just import the component directly instead of getting them from the `dist` folder and making changes in Webpack.